methods/transient/Transient_Requirement_Validation.ipynb

```diff
@@ -0,0 +1,5142 @@
+{
```

Comment:
<p>merge the above 2 code cells into one please.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>